examples/goldilocks_models/find_goldilocks_models.cc

```diff
@@ -61,10 +62,14 @@ DEFINE_int32(N_sample_tr, -1, "Sampling # for turning rate");
 DEFINE_bool(is_zero_touchdown_impact, false,
             "No impact force at fist touchdown");
 DEFINE_bool(is_add_tau_in_cost, true, "Add RoM input in the cost function");
+DEFINE_bool(is_uniform_grid, true, "Uniform grid of task space");
+DEFINE_bool(is_restricted_sample_number, false, "Restrict the number of samples. This makes sense"
+                                               "only when is_uniform_grid=false");
 
 // inner loop
-DEFINE_string(init_file, "", "Initial Guess for Trajectory Optimization. "
-                             "E.g. w0.csv");
+DEFINE_string(init_file, "", "Initial Guess for Trajectory Optimization");
+DEFINE_bool(is_use_interpolated_initial_guess,false,"Use interpolated initial guess"
```

Comment:
`is_use_interpolated_initial_guess` seems redundant. We can use `is_uniform_grid` to tell the inner loop whether or not we want to interpolated initial guess?

Reply:
This makes sense. I will combine these two flags.